cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt

```diff
@@ -51,6 +51,40 @@ import org.ossreviewtoolkit.utils.log
 import java.net.HttpURLConnection
 import java.net.URL
 
+private fun PackageCuration.toContributionPatch(): ContributionPatch {
```

Comment:
Commit message nit: "a**n** extension"
What do you think of putting this function below the class body, so that the first thing you see when opening the file is the expected class, not a private helper function? Because I think inside class bodies we also usually put public functions at the top and private functions at the bottom.

Reply:
> Commit message nit: "a**n** extension"

Done.

> What do you think of putting this function below the class body

I'm fine with that, done.